execution-engine/src/engines/common.rs

```diff
@@ -803,16 +814,14 @@ pub struct WasiWrapper {
     ///            |  Internal    |
     ///            ----------------
     filesystem: FileSystem,
-    /// The environment variables that have been passed to this program from the
-    /// global policy file.  These are stored as a key-value mapping from
-    /// variable name to value.
-    pub(crate) environment_variables: Vec<(String, String)>,
-    /// The array of program arguments that have been passed to this program,
-    /// again from the global policy file.
-    pub(crate) program_arguments: Vec<String>,
-    /// The exit code, if program calls proc_exit.
+    /// The exit code returned by the last executing program.
     exit_code: Option<u32>,
-    /// Whether clock functions (`clock_getres()`, `clock_gettime()`) should be enabled.
+    /// The environment variables currently set, and their bindings.
+    environment_variables: Vec<(String, String)>,
```

Comment:
Why were these two (`environment_variables` and `program_arguments`) moved? It makes the diff hard to read.

Reply:
I moved back to the original order. The git diff is still confused about the change though.